### ISSUE TITLE
Support for array length

### ISSUE THREAD

**Issue by zommerfelds** [vandium-io/lambda-tester#10]
I'm new to Lambda and this library, so maybe this is more of a question than bug report.

When I send an array as my event:
```
return LambdaTester(myHandler)
            .event([{foo: 'bar'}])
            .expectResult(...);
```
My handler receives an object like `{"0":{"foo": "bar"}}` instead of an array. On AWS I can call `.length`, but this is not present when using lambda-tester.

Thanks!

**Comment by dc00p** [comment]
The desired behaviour you are looking for is available in version `2.7.0` of `lambda-tester`.  It supports arrays as events.  See the [CHANGELOG](https://github.com/vandium-io/lambda-tester/blob/2.7.0/CHANGELOG.md).  Please upgrade to the latest release.

**Comment by zommerfelds** [comment]
Oh, thanks for pointing this out! Sorry for the trouble.
